lib/sinon-match-integration.test.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+var assert = require("@sinonjs/referee").assert;
+var samsam = require("./samsam");
+var match = samsam.match;
+
+describe("sinon-match-integration", function() {
+    context("samsam.match", function() {
+        it("returns true if matching custom matcher", function() {
+            var matchAlways = samsam.createMatcher(function() {
+                return true;
+            });
+            var checkMatch = match(42, matchAlways);
+            assert.isTrue(checkMatch);
+        });
+
+        it("returns true if matching boolean", function() {
+            assert.isTrue(samsam.match(false, match.bool));
+        });
+
+        it("returns false if not matching boolean", function() {
+            assert.isFalse(samsam.match(0, match.bool));
+        });
+
+        it("returns true if matching number", function() {
+            assert.isTrue(samsam.match(42, match.number));
+        });
+
+        it("returns false if not matching number", function() {
+            assert.isFalse(samsam.match("42", match.number));
+        });
+
+        it("returns true if matching string", function() {
+            assert.isTrue(samsam.match("abc", match.string));
+        });
+
+        it("returns false if not matching string", function() {
```

Comment:
Small thing and totally preference, but it would be nice to have all the positive and negative tests grouped together under a context.

```js
context('positive matches', ....);

context('negative matches', ....);
```

Possibly with better naming though? Just throwing it out there.

Reply:
Done